offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -227,37 +233,39 @@ class OfferToPDFConverter implements OfferExporter {
     void setSelectedItems() {
         // Let's clear the existing item template content first
         htmlContent.getElementById("product-items-1").empty()
-        //and remove the footer on the first page
-        //htmlContent.getElementById("grid-table-footer").remove()
+        htmlContent.getElementById("product-items-2").empty()
+        // To also remove the styling of the div elements they have to be removed
+        htmlContent.getElementById("grid-table-footer").remove()
```

Comment:
The "grid-table-footer" <div> from the template has to be removed since it introduces a double underscore via css styling. 
Since the `ItemPrintout.tableFooter` method appends a "grid-table-footer" <div> element, just replacing its content will lead to a double double underscoring. because the "grid table footer" <div> will be duplicated.

Reply:
But this should then only happen if you create a new div writing the element, right? I am not sure if I get the problem. Maybe you can show me how that happens? I would assume that the content is not set correctly if then a double double underscore is displayed.

Reply:
I found a solution which keeps the template structure in tact 👍